src/ert/config/ert_config.py

```diff
@@ -372,12 +372,37 @@ def read_site_config(cls) -> ConfigDict:
 
     @classmethod
     def read_user_config(cls, user_config_file: str) -> ConfigDict:
-        site_config = cls.read_site_config()
-        return lark_parse(
-            file=user_config_file,
-            schema=init_user_config_schema(),
-            site_config=site_config,
-        )
+        return lark_parse(user_config_file, schema=init_user_config_schema())
+
+    @classmethod
+    def read_user_config_and_apply_site_config(
+        cls, user_config_file: str
+    ) -> ConfigDict:
+        site_config_dict = cls.read_site_config()
+        user_config_dict = cls.read_user_config(user_config_file)
+
+        for keyword, value in site_config_dict.items():
+            if keyword == "QUEUE_OPTION":
+                filtered_queue_options = []
+                for queue_option in value:
+                    if (
+                        "MAX_RUNNING" in queue_option
```

Comment:
This adds a maintenance problem when we add more queue options later. Can this be automated so we don't need to specifically test for MAX_RUNNING and SUBMIT_SLEEP here?

Reply:
I am not sure, as those are the only options that can be both global and queue specific.

Reply:
I can test if the string is outside of `QUEUE_OPTION`, but I wonder if this can cause other bugs which will be difficult to troubleshoot.